docs/weave-tutorial/06-buckets.md

```diff
@@ -57,64 +61,61 @@ This basically consists of adding _Copy()_ and _Validate()_ to the objects in `c
 
 ## Dive into Code
 
-Let's define `MarketBucket` that will hold `Market` information and write a function that creates a market. This is a basic `morm/model_bucket` without any secondary index.
+Let's define `UserBucket` that will hold `User` information and write a function that creates an user. This is a basic `morm/model_bucket` without any secondary index.
 
 ```go
-type MarketBucket struct {
+type UserBucket struct {
     morm.ModelBucket
 }
 
-func NewMarketBucket() *MarketBucket {
-    b := morm.NewModelBucket("market", &Market{})
-    return &MarketBucket{
-        ModelBucket: b,
+// NewUserBucket returns a new user bucket
+func NewUserBucket() *UserBucket {
+    return &UserBucket{
+        morm.NewModelBucket("user", &User{}),
     }
 }
 ```
 
-Now create your `OrderBookBucket` with secondary indexes. Secondary indexes enable you to insert and query models with ease. Think of this as a SQL index.
+We will demonstrate secondary index usage with `ArticleBucket.`. Secondary indexes enable you to insert and query models with ease. Think of this as a SQL index.
 
 ```go
-type OrderBookBucket struct {
+type ArticleBucket struct {
     morm.ModelBucket
 }
 
-// NewOrderBookBucket initates order book with required indexes
-func NewOrderBookBucket() *OrderBookBucket {
-    b := morm.NewModelBucket("order book", &OrderBook{},
-        morm.WithIndex("market", marketIDindexer, false),
-        morm.WithIndex("marketWithTickers", marketIDTickersIndexer, true),
-    )
-
-    return &OrderBookBucket{
-        ModelBucket: b,
+// NewArticleBucket returns a new article bucket
+func NewArticleBucket() *ArticleBucket {
+    return &ArticleBucket{
+        morm.NewModelBucket("article", &Article{},
+            morm.WithIndex("blog", articleBlogIDIndexer, false),
+            morm.WithIndex("timedBlog", blogTimedIndexer, false)),
     }
 }
 ```
 
-Let's explain what the heck is `morm.WithIndex("market", marketIDindexer, false)`.
+Let's explain what the heck is `morm.WithIndex("blog", articleBlogIDIndexer, false)`?
 
 ## Secondary Indexes
 
-Sometimes we need another index for the data. Generally, we will look up an order book from the market it belongs to and its index in the market. But what if we want to list all Orderbooks of a market overall Order books? For this, we need to add a secondary index on the Order books to query by market. This is a typical case and Weave provides nice support for this functionality.
+Sometimes we need another index for the data. Generally, we will look up an article from the blog it belongs to and its index in the article. But what if we want to list all articles of a blog overall articles? For this, we need to add a secondary index on the articles to query by blog. This is a typical case and Weave provides nice support for this functionality.
```

Comment:
`But what if we want to list all articles of a blog overall articles?`
`I don't understand this sentence.`

Reply:
I will elaborate on that.